lib/components/artist/related_artists/index.js

```diff
@@ -18,23 +20,21 @@ class RelatedArtists extends React.Component {
   }
   
   columnCount() {
-    const isPhone = Dimensions.get('window').width < 700;
-    const isPadHorizontal = Dimensions.get('window').width > 1000;
-    return !isPhone ? isPadHorizontal ? 4 : 3 : 2;
+    const isPhone = window.width < 700;
+    const isPadHorizontal = window.width > 1000;
+    return !isPhone ? (isPadHorizontal ? 4 : 3) : 2;
```

Comment:
In the future it’s better to reduce the amount of negation, as it will be more readable.